mpp/shmem_c_func.h4.in

```diff
@@ -429,14 +429,18 @@ SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_alltoalls64(void *dest, const void *
 
 /* Point-to-Point Synchronization Routines */
 define(`SHMEM_C_WAIT',
-`SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_$1_wait(volatile $2 *var, $2 value);')dnl
-SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_wait(volatile long *ivar, long cmp_value);
+`SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_$1_wait(volatile $2 *var, $2 value) SHMEM_ATTRIBUTE_DEPRECATED;')dnl
+SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_wait(volatile long *ivar, long cmp_value) SHMEM_ATTRIBUTE_DEPRECATED;
 SHMEM_BIND_C_WAIT(`SHMEM_C_WAIT')
 
 define(`SHMEM_C_WAIT_UNTIL',
-`SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_$1_wait_until(volatile $2 *var, int cond, $2 value);')dnl
-SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_wait_until(volatile long *ivar, int cmp, long value);
-SHMEM_BIND_C_WAIT(`SHMEM_C_WAIT_UNTIL')
+`SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_$1_wait_until(volatile $2 *var, shmem_cmp_t cmp, $2 value);')dnl
+SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmem_wait_until(volatile long *ivar, shmem_cmp_t cmp, long value);
```

Comment:
I don't see this `volatile long` signature in the 1.4 spec.  Is it deprecated or just dropped?

Reply:
Perhaps that's what #449 will address...

Reply:
Yup, leaving this for @wrrobin to fix.  :)